index.html

```diff
@@ -7506,7 +7505,7 @@ <h2><code>application/td+json</code> Media Type Registration</h2>
       <p class="ednote" title="Other Fragment Identifier Considerations">
       We may 
       want to explictly allow use of JSON Pointer as a fragment identifier [[RFC6901]].
-      There is in fact an assertion requiring them in TMs for <code>tm:required</code>.
+      There is in fact an assertion requiring them in TMs for <code>tm:optional</code>.
```

Comment:
The fix looks good.
However, I am not 100% sure what the *whole* editors note tries to say 🙈